### DISCUSSION TITLE
Einzelposten bei Sammelüberweisungen

### DISCUSSION THREAD

**Discussion by tolot27** [openjverein/jverein#448] [General]
Wie komme ich denn in JVerein bzw. Hibiscus an die Einzelposten einer Sammelüberweisung ran? In der Buchungsliste sehe ich nur die Sammelüberweisung. Wenn man da nicht an die einzelnen Posten heran kommt, werde ich nie wieder Sammelüberweisungen verwenden, da ich sonst immer manuell Splitbuchungen machen müsste.

**Comment by lenilsas** [comment]
Theoretisch kann man bei der Sammelüberweisung einstellen ob sie als Einzelbuchungen aufgeführt werden sollen, leider unterstützen das die meisten Banken nicht.
Bei JVerein kannst du auf die Sammelbuchung rechtsklick->Split und dann auf Sammelüberweisung, da kannst du die zugehörige Hibiscus Sammelüberweisung auswählen dann werden die Spitbuchungen entsprechend automatisch erstellt.

**Reply by tolot27** [reply]
Danke, dass ist schon mal ein super Hinweis. Ich habe die Sammelüberweisung auch nur gemacht, weil ich es ziemlich mühsam finde, bei mehreren noch nicht ausgeführten Überweisungen einzeln Rechtsklick und dann "Jetzt ausführen..." anzuklicken. Warum ist das ausgegraut, wenn man mehrere Überweisungen selektiert hat?

**Comment by lenilsas** [comment]
Dafür gibt es doch die Synchronisier Funktion von Hibiscus

**Reply by tolot27** [reply]
> Dafür gibt es doch die Synchronisier Funktion von Hibiscus

Du meinst die Option "Fällige SEPA-Überweisungen absenden"? Die hilft aber nur für Fälle, wo das Konto auch gedeckt ist. Nach Veranstaltungen oder größeren Lastschriften muss ich manchmal etwas warten, bis der verfügbare Betrag groß genug ist. Dann schicke ich die Überweisungen los. Klar könnte ich da die Fälligkeit anpassen. Ein klein wenig intuitiver wäre es jedoch, wenn man mehrere Überweisungen auch mit Rechtsklick, "Jetzt ausführen..." wegschicken könnte, so wie es bei einer einzelnen Überweisung auch geht.

**Reply by lenilsas** [reply]
> "Fällige SEPA-Überweisungen absenden"

Die Funktion kenne ich nicht.
Ich meine unter Start->Synchronisierung Starten. Da kannst du auch die Checkbox bei allen Überweisungen entfernen die du noch nicht abschicken möchtest.
Dieses Thema gehört aber zu Hibiscus und nicht nach JVerein.
